apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-env browser */
+/* global TomSelect, MathJax */
+
+window.PLMultipleChoice = function (uuid) {
+  const selectElement = document.getElementById('pl-multiple-choice-select-' + uuid);
+  const container = selectElement.closest('.pl-multiple-choice-dropdown');
+
+  const select = new TomSelect(selectElement, {
+    plugins: ['no_backspace_delete', 'dropdown_input'],
+    allowEmptyOption: true,
+
+    // Search based on the `content` field, which comes from the `data-content`
+    // attribute on each option.
+    searchField: ['content'],
+
+    // Ensure searches happen immediately.
+    refreshThrottle: 0,
+
+    // Mirror default `<select>` behavior, only open on Down/Enter/Space.
+    openOnFocus: false,
+
+    // Render items based on the `data-content` attribute.
+    render: {
+      option: (data) => {
+        return `<div>${data.content}</div>`;
+      },
+      item: (data) => {
+        return `<div class="${data.disabled ? 'text-muted' : ''}">${data.content}</div>`;
+      },
+    },
+  });
+
+  // In case the dropdown items contain math, render it when the
+  // dropdown is opened or closed.
+  select.on('dropdown_open', () => {
+    // The first time the dropdown is opened, this even is fired before the
+    // options are actually present in the DOM. We'll wait for the next tick
+    // to ensure that the options are present.
+    setTimeout(() => MathJax.typesetPromise([container]), 0);
```

Comment:
I made this change when moving this all to a script file; I noticed that math wasn't rendering the first time I opened the dropdown when clicking.